oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -248,6 +248,8 @@ pub struct Runtime {
 
     aux_servers: Mutex<Vec<AuxServer>>,
 
+    introspection_event_sender: broadcast::Sender<Event>,
```

Comment:
comment pls

Reply:
Added! Please let me know if this is the type of comment you were looking for.